rocketpy/tools.py

```diff
@@ -232,32 +232,39 @@ def bilinear_interpolation(x, y, x1, x2, y1, y2, z11, z12, z21, z22):
     ) / ((x2 - x1) * (y2 - y1))
 
 
-def get_distribution(distribution_function_name):
+def get_distribution(distribution_function_name, rng_generator=None):
```

Comment:
Seems like you chose using implicit names rather than a full description.

`rng_generator` -> `random_number_generator`

Seems more readable. What do you think?

Reply:
If you REALLY want to use RNG acronym, I suggest you to re-think if we really need the "_generator" part.

How do you read this variable name? "Random generator number underscore generator"

Reply:
Great point, fixed on [d18408e](https://github.com/RocketPy-Team/RocketPy/pull/649/commits/d18408eb0679f06f89b42a2999b6a3d8c5c5698f).